### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by tonifisler**
- Changed variable name `bootstrapSass` to `bootstrap`
- Change the way we ask if we want to use Contentful (now in the options prompt, with the others)
- Load the latest Boostrap 4 version
- Fix `styleguide.scss` paths to node_modules
- Fix component `name` generation
- Improve the item-component layout
- Fix deployment on `gh-pages`

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -53,23 +53,26 @@ module.exports = yeoman.Base.extend({
...
           name: 'Framework (Bootstrap 4)',
-          value: 'bootstrapSass',
+          value: 'bootstrap',
           checked: true
         }, {
           name: 'Tests (Mocha, Casperjs and Chai)',
           value: 'tests',
           checked: false
+        }, {
+          name: 'Contentful',
```
Comment (Yago) [inline app/index.js]: In this case, what is the purpose of using ContentFull if you are not using the stylguide ? I think we have to keep it inside the condition.

Ex : 
❎ Stylguide
✅ ContentFull

**Review by Yago — COMMENTED** [review]
asdasd

File: app/index.js (hunk 1 above)
Comment (tonifisler) [reply app/index.js]: You're right. I just hate having this question everytime for a feature we have set up once.

What if we just pass the option via a parameter? `$ yo toolbox --contentful`

**Pull request merged**